lib/analyze/collect/metadata.js

```diff
@@ -61,14 +61,12 @@ function normalizeLicense(name, license) {
 
         if (correctedLicense) {
             log.verbose(logPrefix, `Module ${name} license was corrected from ${license} to ${correctedLicense}`);
-            license = correctedLicense;
+            return correctedLicense;
         } else {
             log.verbose(logPrefix, `License for module ${name} is not a valid SPDX indentifier`, { license });
-            license = null;
+            return null;
         }
     }
-
-    return license;
```

Comment:
Huh? That should be functionally identical since `license` is scoped to that function, right? ... It just avoids needing to assign back to `license` at the end.